packages/libs/react-ui/src/components/NavHeader/NavHeader.tsx

```diff
@@ -15,7 +16,7 @@ export type INavItems = INavItem[];
 
 export interface INavHeaderContainerProps {
   brand?: LogoVariant;
-  children?: React.ReactNode;
+  children?: FunctionComponentElement<INavHeaderProps>[];
```

Comment:
hmm is this correct? I believe these props equate to the whole NavHeader object that is getting exported, not each of the individual subcomponents of the NavHeader